model/role.go

```diff
@@ -69,17 +70,25 @@ func init() {
 		PERMISSION_SYSCONSOLE_READ_REPORTING.Id: {
 			PERMISSION_VIEW_TEAM,
 		},
+		PERMISSION_SYSCONSOLE_WRITE_USERMANAGEMENT_USERS.Id: {
+			PERMISSION_EDIT_OTHER_USERS,
```

Comment:
This takes care of the email update? We need a conditional `user.IsSystemAdmin()` on the email update so that system admin emails can't be updated by other roles.

Reply:
my assumption was that, that would be done as part of https://mattermost.atlassian.net/browse/MM-27523

Reply:
Yes I can do it as part of that.

Reply:
Note to @srkgupta this PR _does_ handle the email update.